common/src/main/java/com/revenuecat/purchases/common/BillingWrapper.kt

```diff
@@ -173,9 +180,9 @@ class BillingWrapper(
         presentedOfferingIdentifier: String?
     ) {
         if (replaceSkuInfo != null) {
-            debugLog("Moving from old sku ${replaceSkuInfo.oldPurchase.sku} to sku ${skuDetails.sku}")
+            log(LogIntent.PURCHASE, PurchaseStrings.UPGRADE_SKU.format(replaceSkuInfo.oldPurchase.sku, skuDetails.sku))
         } else {
-            debugLog("Making purchase for sku: ${skuDetails.sku}")
+            log(LogIntent.PURCHASE, PurchaseStrings.PURCHASE_PRODUCT.format(skuDetails.sku))
```

Comment:
This is a clear example of the tense thing - no way to know without going into the strings file whether the string is "purchased" or "making purchase", but those are very different things in this context